circuits/cpp/src/aztec3/circuits/rollup/test_utils/nullifier_tree_testing_harness.hpp

```diff
@@ -35,6 +35,11 @@ class NullifierMemoryTreeTestingHarness : public proof_system::plonk::stdlib::me
     // Current size of the tree
     fr size() { return leaves_.size(); }
 
+    aztec3::circuits::abis::AppendOnlyTreeSnapshot<aztec3::utils::types::NativeTypes> get_snapshot()
```

Comment:
if you import `init.hpp`, then you can just use `NT` directly.

Reply:
Not sure you can by just importing it, think you soulw also need to handle some extra name spacing. Might be better fit with the more general cleanup also addressing @dbanks12 proposal.